packages/editor/src/editor/PortableTextEditor.tsx

```diff
@@ -105,26 +107,46 @@ export class PortableTextEditor extends Component<PortableTextEditorProps> {
    */
   private editable?: EditableAPI
 
+  private store: EditorStore
+
+  private patches$?: PatchObservable
+  private patchesSubscription?: Subscription
+
   constructor(props: PortableTextEditorProps) {
     super(props)
 
     if (!props.schemaType) {
       throw new Error('PortableTextEditor: missing "schemaType" property')
     }
 
-    if (props.incomingPatches$) {
-      console.warn(
-        `The prop 'incomingPatches$' is deprecated and renamed to 'patches$'`,
-      )
-    }
-
     this.change$.next({type: 'loading', isLoading: true})
 
     this.schemaTypes = getPortableTextMemberSchemaTypes(
       props.schemaType.hasOwnProperty('jsonType')
         ? props.schemaType
         : compileType(props.schemaType),
     )
+
+    // If no store is provided then we just create one
+    this.store = props.store ?? createEditorStore()
```

Comment:
The word store here is the only thing I can think could be discussed. I prefer the API – I just don't know about the terminology.

Reply:
Thanks! I've renamed it to `editor` now.